runtime/legion.h

```diff
@@ -2524,6 +2524,7 @@ namespace Legion {
       Task(void);
     public:
       virtual const char* get_task_name(void) const = 0;
+      virtual bool has_trace(void) const = 0;
```

Comment:
Is there a reason that we are making this part of the Task interface? It feels like something mapping specific and not something that should be visible to the application too. I feel like this should be exposed in the mapper interface and not here.